dotnet/content/scmApplicationHost.xdt

```diff
@@ -1,27 +1,10 @@
 ﻿<?xml version="1.0"?>
+<!-- This file exists to prevent applicationHost.xdt from being applied to scm host that runs other dotnet processes (dotnet build, csc etc. )-->
 <configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
-  <!-- This file exists to prevent applicationHost.xdt from being applied to scm host that runs
-      other dotnet processes (dotnet build, csc etc. )-->
-	<environmentVariables xdt:Transform="InsertIfMissing">
-	
-		<!-- These variables enable manual tracing and custom metrics from web jobs in the scm context -->
-        
-        <add name="DD_APM_WINDOWS_PIPE_NAME" xdt:Locator="Match(name)" xdt:Transform="RemoveAll"/>
-        <add name="DD_APM_WINDOWS_PIPE_NAME" value="datadogtrace-uniqueTracePipeId" xdt:Locator="Match(name)" xdt:Transform="Insert"/> <!-- Trace server variable -->
-		
-        <add name="DD_TRACE_PIPE_NAME" xdt:Locator="Match(name)" xdt:Transform="RemoveAll"/>
-        <add name="DD_TRACE_PIPE_NAME" value="datadogtrace-uniqueTracePipeId" xdt:Locator="Match(name)" xdt:Transform="Insert"/> <!-- Trace client variable -->
-
-        <add name="DD_AGENT_PIPE_NAME" xdt:Locator="Match(name)" xdt:Transform="RemoveAll"/>
-        <add name="DD_AGENT_PIPE_NAME" value="dogstatsd-uniqueStatsPipeId" xdt:Locator="Match(name)" xdt:Transform="Insert"/> <!-- Dogstatsd client variable (LEGACY) -->
-		
-        <add name="DD_DOGSTATSD_PIPE_NAME" xdt:Locator="Match(name)" xdt:Transform="RemoveAll"/>
-        <add name="DD_DOGSTATSD_PIPE_NAME" value="dogstatsd-uniqueStatsPipeId" xdt:Locator="Match(name)" xdt:Transform="Insert"/> <!-- Dogstatsd client variable (CURRENT) -->
-		
-        <add name="DD_DOGSTATSD_WINDOWS_PIPE_NAME" xdt:Locator="Match(name)" xdt:Transform="RemoveAll"/>
-        <add name="DD_DOGSTATSD_WINDOWS_PIPE_NAME" value="dogstatsd-uniqueStatsPipeId" xdt:Locator="Match(name)" xdt:Transform="Insert"/> <!-- Dogstatsd server variable -->
-
+  <system.webServer>
+    <runtime xdt:Transform="InsertIfMissing" >
+      <environmentVariables xdt:Transform="InsertIfMissing">
```

Comment:
Why include this if we're not changing anything 🤔

Reply:
This file does transformation on the scm website configuration (scm is the one running kudu and webjobs).
I left this particular entry after iterative tests. It seems adding it don't break the scm site, whereas adding the env variables itself did. So indeed, it's not needed, but as I planned to resume the investigation around web job support, I guessed I could leave it.